content/en/blog/2024/elastic-contributes-continuous-profiling-agent.md

```diff
@@ -0,0 +1,150 @@
+---
+title: Elastic Contributes its Continuous Profiling Agent to OpenTelemetry
```

Comment:
```suggestion
title: Elastic Contributes it's Continuous Profiling Agent to OpenTelemetry
```

Not a native speaker, so I might be wrong. Or drop the "it(')s" altogether, the title _should_ still say the same thing

Reply:
`it's` would mean `it is` ([see here](https://www.merriam-webster.com/grammar/when-to-use-its-vs-its#:~:text=It's%20is%20a%20contraction%20and,%2C%22%20for%20nouns%20without%20gender.)), so `its` should be fine here

Reply:
right... 🙃